core/src/main/java/de/jplag/reporting/csv/comparisons/NameMapperRandomIds.java

```diff
@@ -0,0 +1,44 @@
+package de.jplag.reporting.csv.comparisons;
+
+import java.security.SecureRandom;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Maps the real names of submissions to random ids to anonymize the data.
+ */
+public class NameMapperRandomIds implements NameMapper {
+    private final Map<String, String> map;
+    private final Random random;
+
+    /**
+     * New instance
+     */
+    public NameMapperRandomIds() {
+        this.map = new HashMap<>();
+        this.random = new SecureRandom();
+    }
+
+    private String newId() {
+        String id = String.valueOf(this.random.nextInt(0, Integer.MAX_VALUE));
```

Comment:
Do we actually want to generate numbers of that big o a range?
This might make the csv hard to read